jcl/src/java.base/share/classes/java/lang/System.java

```diff
@@ -138,11 +138,15 @@ public final class System {
 	private static boolean hasSetOutEncoding;
 	private static String consoleDefaultEncoding;
 	/* The consoleDefaultCharset is different from the default console encoding when the encoding
-	 * doesn't exist, or isn't available at startup. Some Charset's are not available in the
-	 * java.base module and so are not used at startup.
+	 * doesn't exist, or isn't available at startup. Some character sets are not available in the
+	 * java.base module, only in the jdk.charsets module, and so are not used at startup.
```

Comment:
nit: Additional character sets can originate from any module; I'll let you decide if that warrants changing this comment.

Reply:
Updated comment.